tests/user.test.js

```diff
@@ -10,6 +11,7 @@ describe('User', () => {
   beforeEach(() => {
     server = createServer();
     api = supertest(server);
+    users.length = 0;
```

Comment:
Directly setting the length of the users array to 0 may not be a good way to ensure a clean test environment.